app/src/organisms/FirmwareUpdateModal/UpdateResultsModal.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react'
+import { useTranslation, Trans } from 'react-i18next'
+import capitalize from 'lodash/capitalize'
+import {
+  ALIGN_CENTER,
+  BORDERS,
+  COLORS,
+  DIRECTION_COLUMN,
+  Flex,
+  Icon,
+  SPACING,
+  TYPOGRAPHY,
+} from '@opentrons/components'
+import { SmallButton } from '../../atoms/buttons'
+import { StyledText } from '../../atoms/text'
+import { Modal } from '../../molecules/Modal'
+
+import type { ModalHeaderBaseProps } from '../../molecules/Modal/types'
+import type { InstrumentData } from '@opentrons/api-client'
+
+interface UpdateResultsModalProps {
+  isSuccess: boolean
+  closeModal: () => void
+  instrument?: InstrumentData
+}
+
+export function UpdateResultsModal(
+  props: UpdateResultsModalProps
+): JSX.Element {
+  const { isSuccess, closeModal, instrument } = props
+  const { i18n, t } = useTranslation(['firmware_update', 'shared'])
+
+  const updateFailedHeader: ModalHeaderBaseProps = {
+    title: t('update_failed'),
+    iconName: 'ot-alert',
+    iconColor: COLORS.red2,
+  }
+
+  return (
+    <>
+      {!isSuccess || instrument?.ok !== true ? (
+        <Modal header={updateFailedHeader}>
+          <Flex flexDirection={DIRECTION_COLUMN}>
+            <StyledText as="p" marginBottom={SPACING.spacing32}>
+              {t('download_logs')}
+            </StyledText>
+            <SmallButton
+              onClick={() => closeModal()}
+              buttonText={i18n.format(t('shared:close'), 'capitalize')}
+              width="100%"
+            />
+          </Flex>
+        </Modal>
+      ) : (
+        <Modal>
+          <Flex
+            flexDirection={DIRECTION_COLUMN}
+            gridGap={SPACING.spacing40}
```

Comment:
![Screenshot 2023-07-18 at 12 27 13 PM](https://github.com/Opentrons/opentrons/assets/474225/7e4911f5-8725-4bb3-b123-e9dc082c6f05)



```suggestion
            gridGap={SPACING.spacing32}
```